src/ansys/mechanical/core/embedding/runtime.py

```diff
@@ -86,4 +101,4 @@ def initialize(version: int) -> None:
         __register_function_codec()
         Logger.debug("Registered function codec")
 
-    _bind_assembly_for_explicit_interface("Ansys.ACT.WB1")
+    _bind_assembly("Ansys.ACT.WB1", True, True)
```

Comment:
let's make it opt-in for now, like this:

`app = App(pep8=True)`

until it has gone through more usage/testing.